lib/stringify.js

```diff
@@ -20,14 +20,20 @@ var defaults = {
     delimiter: '&',
     encode: true,
     encoder: utils.encode,
+    format: 'RFC3986',
+    formatters: {
+        /** @return {string} */
```

Comment:
What's wrong with jsdoc? It's very helpful with IDE code autocompletion and code proofing.